lib/link_state.h

```diff
@@ -160,6 +169,8 @@ struct ls_node {
 #define LS_ATTR_ADJ_SID6	0x04000000
 #define LS_ATTR_BCK_ADJ_SID6	0x08000000
 #define LS_ATTR_SRLG		0x10000000
+#define LS_ATTR_ADJ_SRV6SID	0x20000000
+#define LS_ATTR_BCK_ADJ_SRV6SID 0x40000000
 #define LS_ATTR_EXT_ADM_GRP 0x20000000
```

Comment:
Be careful. You define twice the 0x2000000 flag: For Extended Admin Group and then for Adjacency SRv6 SID. You need to increment (shift by 1 bit position) the flag to avoid problem when parsing the stream.